docs/content/latest/api/ysql/the-sql-language/statements/ddl_alter_table.md

```diff
@@ -76,7 +88,112 @@ Renaming a table is a non blocking metadata change operation.
 
 Drop the named column from the table. 
 
-- `RESTRICT` — Remove only the specified
+- `RESTRICT` — Remove only the specified column
+- `CASCADE` — Remove the specified column and any dependent objects.
+
+##### Example
+
+Set up a and populate a parents-children pair of tables thus:
+
+```plpgssql
+drop table if exists children cascade;
+drop table if exists parents  cascade;
+
+-- The column "b" models a (natural) business unique key.
+create table parents(
+  k int primary key,
+  b int not null,
+  v text not null,
+  constraint parents_b_unq unique(b));
+
+create table children(
+  parents_b  int  not null,
+  k         int  not null,
+  v         text not null,
```

Comment:
Here it is: the world's nit-pickiest nitpick. :D
```suggestion
  k          int  not null,
  v          text not null,
```

Reply:
Done. Thanks.